Detections/AzureDevOps/AzDOAdminGroupAdditions.yaml

```diff
@@ -19,7 +19,7 @@ relevantTechniques:
 query: |
 
   let timeframe = 4h;
-  // Monitor for Project Administrator adds to *any* project
+  // Change to true to monitor for Project Administrator adds to *any* project
   let MontiorAllProjects = false;
   // If MonitorAllProjects is false, trigger only on Project Administrator add for the following projects
   let ProjectsToMonitor = datatable(ProjectName:string)
```

Comment:
Since this is single dimensional array, suggest changing to dynamic([]) instead of datatable for simplicity.

Reply:
Changed